pkg/terminal/colorize/colorize_test.go

```diff
@@ -0,0 +1,92 @@
+
+func TestPrint(t *testing.T) {
+	dat, err := ioutil.ReadFile(filepath.Join(test.FindFixturesDir(), "issue2897.go"))
+	if err != nil {
+		t.Fatalf("read test fixture error: %v", err)
+	}
+
+	// ensures the AST analysis behaves as expected,
+	// please update `printed` if `terminal/colorize.Print` changes.
+	buf := &bytes.Buffer{}
+	colorize.Print(buf, "main.go", bytes.NewBuffer(dat), 1, 30, 10, colors)
+
+	colorize.Print(os.Stdout, "main.go", bytes.NewBuffer(dat), 1, 30, 10, colors)
```

Comment:
Either remove this line or guard it with a check:

```
const printToStdout = false
if printToStdout {
    colorize.Print(os.Stdout, ...)
}
```

Reply:
done